src/convert/mapEvent.js

```diff
@@ -13,15 +13,26 @@ const { toSlatePath } = require('../utils/convert');
  * mapEvent(event: Y.YMapEvent<any>): SetNodeOperation[]
  */
 const mapEvent = (event) => {
+  
+  /**
+   * convertMapOp(targetElement: event.target, key: string): json | string
+   */
+  const convertChildToSlate = (targetElement, key)=>{
```

Comment:
spaces around "=>" to match style elsewhere?

Reply:
Spaces were added to match the existent style.